### PR TITLE
fedora: Bump OVN version to 21.12.0 and fedora version to 35.

### PR DESCRIPTION
**Pull request by numansiddique**
Signed-off-by: Numan Siddique <numans@ovn.org>

**- What this PR does and why is it needed**

**- Special notes for reviewers**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: dist/images/Dockerfile.fedora [hunk 1]
```diff
@@ -9,13 +9,13 @@
 # are built locally and included in the image (instead of the rpm)
 #
 
-FROM fedora:34
+FROM fedora:35
```
Comment (andreaskaris) [inline dist/images/Dockerfile.fedora:12]: test (ignore the noise)

**Review by andreaskaris — APPROVED** [review]

**Comment by dougsland** [comment]
/lgtm 

See Also: https://koji.fedoraproject.org/koji/buildinfo?buildID=1903456

**Pull request merged**